test/Microsoft.AspNetCore.SignalR.Tests/DefaultHubLifetimeManagerTests.cs

```diff
@@ -110,54 +110,6 @@ public class DefaultHubLifetimeManagerTests
             }
         }
 
-        [Fact]
-        public async Task SendConnectionAsyncDoesNotThrowIfConnectionFailsToWrite()
-        {
-            using (var client = new TestClient())
-            {
-                var manager = new DefaultHubLifetimeManager<MyHub>(new Logger<DefaultHubLifetimeManager<MyHub>>(NullLoggerFactory.Instance));
-
-                var connectionMock = HubConnectionContextUtils.CreateMock(client.Connection);
-                // Force an exception when writing to connection
-                connectionMock.Setup(m => m.WriteAsync(It.IsAny<HubMessage>())).Throws(new Exception("Message"));
-                var connection = connectionMock.Object;
-
-                await manager.OnConnectedAsync(connection).OrTimeout();
-
-                await manager.SendConnectionAsync(connection.ConnectionId, "Hello", new object[] { "World" }).OrTimeout();
-            }
-        }
-
-        [Fact]
-        public async Task SendAllAsyncSendsToAllConnectionsEvenWhenSomeFailToSend()
```

Comment:
???

Reply:
The contract is that HubConnectionContext.WriteAsync never throws.

Reply:
Which is what the test was testing

Reply:
We've already regressed this behavior before, would be nice if we had a test to make sure we don't do it again

Reply:
Nobody **ever** breaks a contract. That's like a solemn vow between engineers.

Reply:
I need to update Redis as well but I left it as is for now

Reply:
> We've already regressed this behavior before, would be nice if we had a test to make sure we don't do it again

What do you suggest? I can make HubConnectionContext.WriteAsync non virtual.